src/pip/_internal/models/direct_url.py

```diff
@@ -22,8 +22,9 @@ class DirectUrlValidationError(Exception):
     pass
 
 
-def _get(d, expected_type, key, default=None):
-    # type: (Dict[str, Any], Type[T], str, Optional[T]) -> Optional[T]
+def _get(
+    d: Dict[str, str], expected_type: Type[T], key: str, default: Optional[T] = None
+) -> Optional[T]:
```

Comment:
The type of `d` changed?

Reply:
Oh, I think I made a mistake. I'll fix it.